apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -52,11 +52,11 @@ function showHomepage() {
   }
 
   // Default teacher announcement.
-  let announcementHeading = i18n.announcementHeadingCoursesEFImprovements();
-  let announcementDescription = i18n.announcementDescriptionCoursesEFImprovements();
+  let announcementHeading = i18n.announcementHeadingHOCiscoming();
+  let announcementDescription = i18n.announcementDescriptionHOCiscoming();
```

Comment:
`i18n.announcementDescriptionHOCiscoming()` strings don't exist...? I think you can just use `i18n.announcementHeading()` since that's the label for the string you want in the i18n file. Same for description.